Provider/src/FirebirdSql.Data.FirebirdClient/Schema/FbSchema.cs

```diff
@@ -35,6 +36,7 @@ internal abstract class FbSchema
 
 		public FbSchema()
 		{
+			MajorVersionNumber = 0;
```

Comment:
`0` is default anyway, why bother?

Reply:
I'll remove it, the ctor (which was already there) is empty after that and can be removed too btw.

Reply:
Agreed. Feel free to remove it too.

Reply:
Done :)